gradlew.bat

```diff
@@ -1,90 +1,90 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
You have edited these files. That's why gradle build is failing in Travis

Reply:
@CloudyPadmal can you please help me how to solve this because there is no difference between the gradlew.bat file before and after this commit as you can see in the diff also I never made any changes in this file while fixing this error . Thank You.